OSBarcodeLib.podspec

```diff
@@ -0,0 +1,13 @@
+
+Pod::Spec.new do |spec|
+  spec.name = "OSBarcodeLib"
+  spec.version = "0.0.1"
+  spec.summary = "The OSBarcodeLib is a library built using Swift that offers you a barcode scanner for your iOS application."
+  spec.homepage = "https://github.com/OutSystems/OSBarcodeLib-iOS"
+  spec.license = "MIT"
+  spec.author = "OutSystems"
```

Comment:
Not sure about this, but maybe we could reference this to ourselves? `s.author = { 'Mobile Ecosystem Team' => 'rd.mobileecosystem.team@outsystems.com' }`

Reply:
I made it `{ 'Outsystems Mobile Ecosystem Team' => 'rd.mobileecosystem.team@outsystems.com' }`